### PR TITLE
Use @nimiq/style v0.5 SVG sprite for icons

### PR DESCRIPTION
**Pull request by sisou**
Everywhere, except in Export, as Sebastian is doing that in his PR himself.

### CODE REVIEW

File: src/common.css [hunk 1]
```diff
@@ -57,12 +57,17 @@
     left: 4rem;
     cursor: pointer;
     opacity: 0.4;
-    transition: opacity .3s ease, left .4s ease;
+    transition: opacity .3s ease, left .3s ease;
+}
+
+.page-header-back-button svg {
```
Comment (nibhar) [inline src/common.css]:
```suggestion
.page-header-back-button .nq-icon {
```

File: src/components/PassphraseBox.css [hunk 2]
```diff
@@ -7,22 +7,28 @@
...
+    opacity: 0.5;
+    width: 3rem;
+    height: 3rem;
     background-size: 3.75rem;
     z-index: 100;
     cursor: pointer;
+    transition: opacity .3s ease;
+}
+
+.passphrase-box .cancel svg {
```
Comment (nibhar) [inline src/components/PassphraseBox.css]:
```suggestion
.passphrase-box .cancel .nq-icon {
```

File: src/request/change-passphrase/ChangePassphrase.css [hunk 3]
```diff
@@ -1,11 +1,8 @@
-.walletfile-locked {
+.page#enter-passphrase .lock {
     display: block;
-    width: 9rem;
-    height: 11.75rem;
+    width: 8rem;
+    height: 11.52875rem;
```
Comment (nibhar) [inline src/request/change-passphrase/ChangePassphrase.css]: Where does that number come from?

Comment (sisou) [reply src/request/change-passphrase/ChangePassphrase.css]: Looks like a typo :D

**Pull request merged**